package.json

```diff
@@ -340,7 +340,9 @@
     "onCommand:csharp.downloadDebugger",
     "onCommand:csharp.listProcess",
     "onCommand:csharp.listRemoteProcess",
-    "workspaceContains:project.json"
+    "workspaceContains:project.json",
+    "workspaceContains:*.csproj",
```

Comment:
Ah -- personally I would definitely want that. We typically open vscode to the root of the repository, and it doesn't have any projects or anything in it -- they are all nested under sub-folders. What are the cases you are thinking about where you wouldn't want that behavior?